Google Scholar.js

```diff
@@ -828,7 +844,11 @@ ItemFactory.prototype.getCourt = function () {
 	if (m) {
 		this.v.court = m[2].replace(/_/g, " ");
 		if (m[1]) {
-			this.v.extra = "{:jurisdiction: " + m[1] + "}";
+			if (!Zotero.Utilities.setMultiField) {
+				this.v.jurisdiction = m[1];
```

Comment:
D'oh.

For the rest, good suggestions as always. I think the second option should work, and anything that brings MLZ and Zotero play nicer together is a  good thing.

MLZ stores weird field data in `extra` in an encoded form during sync (JSON, with a header to signal the offset), and that rigamarole shouldn't be replicated in the translator. Parsing out field content on translator import should work fine. It's a little tricky, though, since the field name in the `{:field-name: XXX}` markup is a CSL variable, and may need to be reverse-mapped into the database schema in some cases. MLZ has a function for that already, though (for reconstructing missing items from embedded document data), and it should be easy enough to tie it into the translator processing chain. I'll take a look.